templates/newsDescription.hbs

```diff
@@ -0,0 +1,21 @@
+<section class="description__container">
+    <div class="container">
+        <div class="row">
+          <div class="description col-12 col-sm-12 col-lg-12">
+              <div class="description__image">
+                  {{{asset image}}}
+                  <div class="description__block"></div>
+              </div>
+              <div class="description__text">
+                 <div class="description__line"></div>
```

Comment:
I think this `div` does not add anything to the componente. I guess you can safely delete it

Reply:
Done!
It was a copy/paste of a previous component, sorry! Removed.